styles.css

```diff
@@ -0,0 +1,343 @@
+  flex-wrap: wrap;
+  /* max-width: 25%; */
+}
+
+.theme-switch-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 0.5rem 0;
+}
+
+.theme-switch {
+  display: inline-block;
+  height: 34px;
+  position: relative;
+  width: 60px;
+}
+
+.theme-switch input {
+  display: none;
+}
+
+.slider {
+  background-color: var(--tertiary-bg);
+  bottom: 0;
+  cursor: pointer;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: 0.4s;
+}
+
+.slider:before {
+  background-color: var(--button);
+  bottom: 4px;
+  content: "";
+  height: 26px;
+  left: 4px;
+  position: absolute;
+  transition: 0.4s;
+  width: 26px;
+}
+
+input:checked + .slider {
+  background-color: var(--hover-bg);
+}
+
+input:checked + .slider:before {
+  transform: translateX(26px);
+  content: "";
+}
+
+.slider.round {
+  border-radius: 34px;
+}
+
+.slider.round:before {
+  border-radius: 50%;
+}
+
+.empty-state {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.empty-state > img {
+  width: 12rem;
+  height: 12rem;
+  opacity: 0.1;
+}
+
+.empty-state > h2 {
+  margin: 3rem 0 0.5rem 0;
+  font-size: 2rem;
+  opacity: 0.2;
+  color: var(--secondary-text);
+}
+
+.empty-state > p {
+  opacity: 0.3;
+  color: var(--secondary-text);
+}
+
+.left {
+  width: 25%;
+  min-width: 100px;
+  padding: 2rem;
+  background-color: var(--secondary-bg);
+  overflow-y: scroll;
+  transition: background-color 0.25s ease-in;
+}
+
+.notes-list {
+  margin: 2rem auto;
+  color: var(--secondary-text);
+}
+
+.notes-list-heading {
+  font-weight: 700;
+  font-size: 1.5rem;
+  letter-spacing: 0.1rem;
+  color: var(--primary-text);
+  text-align: center;
+}
+
+.note-list-item,
+.note-list-item-selected {
+  padding: 1rem;
+  margin: 0.5rem -0.8rem;
+  cursor: pointer;
+  border-radius: 1rem;
+  transition: color, background-color 0.3s ease-in-out;
+  color: var(--primary-text);
+}
+
+.note-list-item:hover {
+  background-color: var(--hover-bg);
+  color: var(--button);
+}
+
+.note-list-item-title {
+  font-size: 1.2rem;
+  font-weight: bold;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 1;
+  -webkit-box-orient: vertical;
+}
+
+.note-list-item-note {
+  font-size: 1rem;
+  color: var(--secondary-text);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  margin-top: 0.5rem;
+}
+
+.note-list-item-selected {
+  background-color: var(--tertiary-bg);
+  color: var(--button);
+}
+
+.note-list-item-selected .note-list-item-note {
+  color: var(--secondary-text);
+}
+
+.btn {
+  padding: 1rem 0;
+  width: 100%;
+  box-shadow: none;
+  border-radius: 2rem;
+  font-size: 1rem;
+  font-weight: 700;
+  letter-spacing: 0.1rem;
+  cursor: pointer;
+  text-transform: uppercase;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.add {
```

Comment:
class name could be better...to describe it more. Helps aboid future conflicts & code readability